db/schema.rb

```diff
@@ -494,7 +494,7 @@
     t.bigint "school_id", null: false
     t.string "name", null: false
     t.string "device_identifier", null: false
-    t.boolean "active", default: false
+    t.boolean "active", default: true
```

Comment:
Is this an issue with your local schema? A CAD device should be inactive by default I think?